firmware/network/border_router/border_router.c

```diff
@@ -36,6 +39,25 @@ int8_t get_wired_iface(void) {
     return -1;
 }
 
+int8_t get_wireless_iface(void) {
+    int max_ifaces = gnrc_netif_numof();
+    if (max_ifaces > 0) {
+        gnrc_netif_t *iface;
+         for (uint8_t i = 0; i < ARRAY_SIZE(radio_devices); i++) {
+            iface = gnrc_netif_get_by_type(radio_devices[i], NETDEV_INDEX_ANY);
+            if (iface != NULL) {
+                break;
+            }
+        }
+        if (iface != NULL) {
+            return iface->pid;
+        } else {
+            return -1;
+        }
+    }
+    return -1;
+}
+
 int border_router_add_ipv6(int cast_type, ipv6_addr_t *addr) {
```

Comment:
This function and the border_router_add_ipv6_node() I see that do the same, This could implement as generic function, that could take both types of network devices